internal/services/compute/windows_virtual_machine_resource_other_test.go

```diff
@@ -483,14 +483,14 @@ func TestAccWindowsVirtualMachine_otherUserData(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.ImportStep("admin_password"),
+		data.ImportStep("admin_password", "user_data"),
```

Comment:
(as elsewhere) can we load the UserData from the API by setting the Option:

```suggestion
		data.ImportStep("admin_password"),
```